servicebox-app/src/main/java/io/halkyon/model/Claim.java

```diff
@@ -43,9 +49,17 @@ public Claim(String name, String serviceRequested, String description, String st
     public String status;
     @FormParam
     public String owner;
-
     @FormParam
     public Date created;
+    @OneToOne(cascade = CascadeType.ALL)
+    public ClaimingStatus claimingStatus;
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(
+            name = "Claim_Service",
+            joinColumns = { @JoinColumn(name = "claim_id") },
+            inverseJoinColumns = { @JoinColumn(name = "Service_id") }
+    )
+    public Set<Service> linkedServices = new HashSet<>();
```

Comment:
It's not clair for me the difference between the `requestedService`and `linkedService`

Reply:
Maybe, we should refactor the `requestedService` and `linkedService` in a single relationship using a new table:
ClaimService with claim ID, requested service, status, attempts. 
Wdyt?

Reply:
As shared together let's start by having a one to one relationship and let the fields in the Claim entity and will scale up later.

Reply:
I've removed the `linkedServices` field and created a one-to-one field called `service`.